operators/tensorboard-controller/requirements.txt

```diff
@@ -0,0 +1,2 @@
+ops >= 1.2.0
+oci-image==1.0.0
```

Comment:
For repos we control, do we want to pin to a specific version or give a range (similar to the sdi repo)?  Until we've got a better way of updating stuff like this automatically, I'm fearing when we bug release something for `oci-image==1.0.0` and need to do 20 charm updates